webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.scss

```diff
@@ -0,0 +1,7 @@
+.inventory_title {
```

Comment:
Do we still need it? I don't see you use it anywhere

Reply:
You're right, it's not used, deleted :tada: